atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/TableSplittingKeyValueService.java

```diff
@@ -377,8 +378,7 @@ public void compactInternally(TableReference tableRef, boolean inMaintenanceMode
     public ClusterAvailabilityStatus getClusterAvailabilityStatus() {
         return delegates.stream()
                 .map(kvs -> kvs.getClusterAvailabilityStatus())
-                .sorted()
-                .findFirst()
+                .min(Comparator.naturalOrder())
```

Comment:
nice

Reply:
This was an automated gradle-baseline refactor